### DISCUSSION TITLE
System backup

### DISCUSSION THREAD

**Discussion by smala777** [roleoroleo/yi-hack-MStar#330] [Q&A]
Hi. I have a chinise clone for YI camera (dont know a model) , can i take backup with any scrips?

**Comment by roleoroleo** [comment]
Is it based on the MStar platform?

**Comment by smala777** [comment]
Hi. I don't know(( can you help me with identification?

сб, 20 бер. 2021, 20:28 користувач roleo ***@***.***> пише:

**Comment by roleoroleo** [comment]
You have to open it and take some pictures.
